bom/pom.xml

```diff
@@ -159,6 +159,7 @@
         <!-- Used for integration tests, to make sure webjars work-->
         <bootstrap.version>3.1.0</bootstrap.version>
         <keycloak.version>6.0.1</keycloak.version>
+        <subethasmtp.version>3.1.7</subethasmtp.version>
```

Comment:
Is this needed in the BOM since it's only for testing?

Reply:
Others test dependencies are also there. I've followed the pattern.